.github/workflows/gradle.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: "🔧 Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1.1.4
         with:
-          distribution: 'graalvm-community'
```

Comment:
Maybe we should replace GraalVM with some other java distribution here since we don't execute any native tests in `gradle` github workflow. All native tests are executed in `graalvm-latest` and `graalvm-dev` github workflows. Your thoughts?

Reply:
make sense to me.